release/nightly_tests/dataset/data_ingest_benchmark_compute_gpu.yaml

```diff
@@ -0,0 +1,15 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 3
+
+head_node_type:
+    name: head_node
+    instance_type: m5.2xlarge
```

Comment:
We use `m5.4xlarge` for `data_ingest_benchmark_compute.yaml`. Any reason why we're using a smaller instance here?

Reply:
This test case is smaller. Less data and num workers.